src/content/blog/astro-560.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Astro 5.6'
+description: 'Astro 5.6 brings first class astro:env and experimental session support to Cloudflare, and gives more control over prefetching.'
+homepageLink:
+  title: 'Astro 5.6'
+  subtitle: 'Available now!'
+publishDate: '2025-04-03'
+authors:
+  - matt
+coverImage: '/src/content/blog/_images/astro-560/blog-post-5-6.webp'
+socialImage: '/src/content/blog/_images/astro-560/og-astro-5-6.webp'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+
+**Astro 5.6 brings first class astro:env and experimental session support to Cloudflare, and gives more control over prefetching.**
+
+☁️ Soar through the clouds with these new features in Astro:
+
+- [**Global `astro:env` on Cloudflare**](#global-astroenv-on-cloudflare)
+- [**Experimental sessions on Cloudflare**](#experimental-sessions-on-cloudflare)
+- [**New prefetch eagerness option**](#new-prefetch-eagerness-option)
+- [**Improved config validation**](#improved-config-validation)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Global `astro:env` on Cloudflare
+
+Since Astro 5 you have been able to access [type-safe environment variables using `astro:env`](https://docs.astro.build/en/guides/environment-variables/#type-safe-environment-variables). However, due to limitations in the Cloudflare platform, until now environment variables in Cloudflare were only accessible within a request — limiting when and where you could use them. For example, if you wanted to use a shared API client instance you couldn't use environment variables if instantiating outside of a request. This could be confusing, and was different from other adapters. Astro 5.6 lifts this restriction, meaning you can now access your environment variables globally throughout your server code, bringing the Cloudflare adapter in line with other official adapters. This is possible thanks to [enhancements in Cloudflare Workers](https://developers.cloudflare.com/changelog/2025-03-17-importable-env/) that are now implemented in Astro.
+
+```js
+import { defineMiddleware } from 'astro:middleware';
+import { API_URL } from 'astro:env/server';
+import { createClient } from './client.js';
+
+// Astro 5.5: undefined
+// Astro 5.6: string
+const client = createClient(API_URL);
+
+export const onRequest = defineMiddleware((ctx, next) => {
+	ctx.locals.client = client;
+	return next();
+});
+```
+
+This improvement creates a more consistent developer experience across all Astro server environments.
```

Comment:
non-blocking nit: Feels like this ideally wants a second sentence here for flow. Could even be light and humourous!